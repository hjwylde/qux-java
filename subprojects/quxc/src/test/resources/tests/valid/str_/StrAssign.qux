str f(str foo):
    foo[0] = 'qux'

    return foo

void main([str] args):
    foo = 'bar'
    print foo[0]
    print foo[1]
    print foo[2]
    print f(foo)
    print f(foo)[0]
    print foo[0]
    foo[1] = {1, 2}
    print foo

