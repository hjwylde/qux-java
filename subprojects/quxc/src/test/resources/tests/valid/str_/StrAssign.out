b
a
r
quxar
q
b
b{1, 2}r
